### DISCUSSION TITLE
Slack channel for maintainers (and others who want to participate)

### DISCUSSION THREAD

**Discussion by jsamoocha** [hozn/stravalib#401] [General]
Hi all,

I created a Slack channel to discuss matters that might be off-topic regarding issues or pull requests and a way to send direct messages to one another.

For anyone interested, you can join using [this invite link](https://join.slack.com/t/stravalibmaintainers/shared_invite/zt-21amocuv4-zECnv3bT~jheD8PeIRWmKQ).

**Comment by lwasser** [comment]
thank you for doing that, @jsamoocha !!
